elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore.rb

```diff
@@ -105,6 +105,16 @@ def __build_connections
               :selector => options[:selector]
           end
 
+          # Closes all connections by marking them as dead
+          # and closing the underlying HttpClient instances
+          #
+          # @return [Connections::Collection]
+          #
+          def __close_connections
+            super
```

Comment:
indeed this only makes sense along with the "marking connections as dead" change above. if this goes away then :+1: on your suggestion